### PR TITLE
fixed issue #8970

### PR DESCRIPTION
**Pull request by shivendra-webkul**
## Issue Reference
Fixed issue #8970
Fixed issue #8969

### CODE REVIEW

**Comment by Amitk-webkul** [comment]
@shivendra-webkul To calendar is going down. it should be the same as the design. [Here](https://prnt.sc/4e7CaqH6tu17)

**Comment by shivendra-webkul** [comment]
@Amitk-webkul Input data is longer in date time format so its overlap on the calendar icon so I break the input fields its not an issue.

**Comment by Amitk-webkul** [comment]
@shivendra-webkul Okay, functionality is working fine.

**Review by jitendra-webkul — APPROVED** [review]

File: packages/Webkul/Admin/src/Resources/views/marketing/promotions/cart-rules/create.blade.php [hunk 1]
```diff
@@ -702,14 +702,13 @@ class="!text-[14px] !text-gray-600 dark:!text-gray-300 font-semibold cursor-poin
...
                             
                                 <x-slot:content>
-                                    <div class="flex gap-[16px]">
                                         <x-admin::form.control-group class="mb-[10px]">
                                             <x-admin::form.control-group.label>
                                                 @lang('admin::app.marketing.promotions.cart-rules.create.from')
                                             </x-admin::form.control-group.label>
         
                                             <x-admin::form.control-group.control
-                                                type="date"
```
Comment (jitendra-webkul) [inline packages/Webkul/Admin/src/Resources/views/marketing/promotions/cart-rules/create.blade.php:712]: This can not be datetime

**Comment by ashishkumar-webkul** [comment]
@shivendra-webkul  pelase check the PR again.

**Pull request merged**